### DISCUSSION TITLE
silent install with parameter lang did not work

### DISCUSSION THREAD

**Discussion by lohnsklave** [d2phap/ImageGlass#322] [General] [answered]
I have a question about silent installation of imageglass. When I install the exe with parameters /VERYSILENT /LANG=GERMAN, this did not work. The settings remain in English.

I also look for an registry entry but did not find anything. Where are the language and the other settings stored? When I change the language manually, it is working fine. But this is not practical for me.

Also another problem is, that I must change the settings for each windows user, is there another option, that I can do the settings for system? 

Thank you very much, Stefan

**Comment by d2phap** [comment, marked as answer]
Hi Stefan,

ImageGlass is packed by Inno Setup, you can look for the commands here: 
http://www.jrsoftware.org/ishelp/index.php?topic=setupcmdline
These commands are for the setup only, not the ImageGlass.

### To change ImageGlass settings,
* If ImageGlass is installed in Program File folder, it will write settings to registry:
`Computer\HKEY_CURRENT_USER\Software\PhapSoftware\ImageGlass`

* If ImageGlass is installed in other folder which has write access (PORTABLE MODE), it will write settings to `igconfig.xml` file.

You can check if the portable mode is enabled or not by going to `Settings > General > Portable mode`.



### To change language,
You give the full path of `*.iglang` file to `Language` key
```xml
<Item key="Language" value="D:\DEV\ImageGlass\Source\ImageGlass\bin\Release\Languages\Vietnamese.iglang" />
```

Hope this will help!
Phap
